src/main/java/org/ambraproject/wombat/controller/ArticleMetadata.java

```diff
@@ -227,12 +227,16 @@ RevisionMenu getRevisionMenu() throws IOException {
   /**
    * Get the articleItems that represent peer review decisions and responses.
    */
-  List<Map<String,?>> getPeerReview() {
-    List tprItems = itemTable.values().stream()
+  String getPeerReview() {
```

Comment:
A String should be sufficient for now, but when we put a real UI in the tab we'll need to change this.